getting_started/step_by_step/your_first_game.rst

```diff
@@ -241,23 +240,23 @@ and use ``clamp()`` to prevent it from leaving the screen:
         position.y = clamp(position.y, 0, screensize.y)
 
 
-.. tip:: *Clamping* a value means restricting it to a given minimum/maximum range.
+.. tip:: *Clamping* a value means restricting it to a given range.
 
-Click "Play the Edited Scene. (F6)" and confirm you can move the player
+Click "Play the edited scene" (``F6``) and confirm you can move the player
 around the screen in all directions.
 
 .. warning:: If you get an error in the "Debugger" panel that refers to a "null instance",
-             this likely means you spelled the node name wrong. Node names are case sensitive
+             this likely means you spelled the node name wrong. Node names are case-sensitive
              and ``$NodeName`` or ``get_node("NodeName")`` must match the name you see in the scene tree.
 
 Choosing Animations
 ~~~~~~~~~~~~~~~~~~~
 
 Now that the player can move, we need to change which animation the
-AnimatedSprite is playing based on direction. We have a "right"
-animation, which should be flipped horizontally (using the ``flip_h``
-property) for left movement, and an "up" animation, which should be
-flipped vertically (``flip_v``) for downward movement.
+AnimatedSprite is playing based on direction. We have a ``right``
+animation, which should be flipped horizontally using the ``flip_h``
+property for left movement, and an ``up`` animation, which should be
+flipped vertically with ``flip_v`` for downward movement.
 Let's place this code at the end of our ``_process()`` function:
```

Comment:
I feel like we're getting inconsistent with what gets "quotes" and what gets ``code tags``. On line 41 you changed the "Width" and "Height" properties to use quotes, but here you changed the "up" animation name to use code tags. Recently, I've been trying to stick to a standard:

- code tags:
Values, variable and function names, Node types

- quotes
Property names, menu items